packages/contracts-bedrock/test/DeploySuperchain.t.sol

```diff
@@ -165,6 +176,35 @@ contract DeploySuperchainOutput_Test is Test {
         vm.expectRevert(expectedErr);
         dso.protocolVersionsProxy();
     }
+
+    function test_writeOutputFile_succeeds() public {
+        DeploySuperchainOutput.Output memory output = DeploySuperchainOutput.Output({
+            superchainProxyAdmin: ProxyAdmin(makeAddr("superchainProxyAdmin")),
+            superchainConfigImpl: SuperchainConfig(makeAddr("superchainConfigImpl")),
+            superchainConfigProxy: SuperchainConfig(makeAddr("superchainConfigProxy")),
+            protocolVersionsImpl: ProtocolVersions(makeAddr("protocolVersionsImpl")),
+            protocolVersionsProxy: ProtocolVersions(makeAddr("protocolVersionsProxy"))
+        });
+
+        dso.set(dso.superchainProxyAdmin.selector, address(output.superchainProxyAdmin));
+        dso.set(dso.superchainConfigImpl.selector, address(output.superchainConfigImpl));
+        dso.set(dso.superchainConfigProxy.selector, address(output.superchainConfigProxy));
+        dso.set(dso.protocolVersionsImpl.selector, address(output.protocolVersionsImpl));
+        dso.set(dso.protocolVersionsProxy.selector, address(output.protocolVersionsProxy));
+
+        string memory root = vm.projectRoot();
+        string memory path = string.concat(root, "/.testdata/test-deploy-superchain-output.toml");
+        dso.writeOutputFile(path);
+        string memory data = vm.readFile(path);
+
+        // Clean up before asserting so that we don't leave any files behind.
+        vm.removeFile(path);
+
+        assertEq(
+            keccak256(abi.encode(data)),
+            bytes32(0x496d8cafe47414684c9ccf13231a39c825f3740e0ed19854c54e1df691591fbd)
```

Comment:
How was this magic value obtained? Any way we can compute this within solidity?

Reply:
I generated the file, checked its value by hand, then hashed it. I can update it to use a local fixture.

Reply:
Updated to use local fixture and compare content rather than a hash.